### ISSUE TITLE
Instructions doesn't match the test

### ISSUE THREAD

**Issue by neilhan** [exercism/purescript#224]
https://github.com/exercism/purescript/blob/bd3c772ec6da6f2d1d018ce36c337ced984102ad/exercises/practice/bob/test/Main.purs#L42
Instruction says:
"""
He answers 'Calm down, I know what I'm doing!' if you yell a question at him.
"""
In the test, the assert was incorrect. It should be:
```
test "forceful questions" do
      Assert.equal "Calm down, I know what I'm doing!" $
        Bob.hey "WHAT THE HELL WERE YOU THINKING?"
```

**Comment by emiel** [comment]
@ErikSchierboom : This issue has been solved by #219 and may now be closed. Thanks!

**Comment by ErikSchierboom** [comment]
Thanks
